src/Components/Header/Header.module.scss

```diff
@@ -0,0 +1,67 @@
+@import '../../style/variables.scss';
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  height: 20%;
+  padding: 1em;
+  background-color: $midBlue;
+  box-shadow: 0px 0px 20px $darkBlue;
+}
+
+.title {
+  text-decoration: none;
+  font-family: $primaryFont;
+  font-size: 2em;
+  color: $lightTan;
+  margin: 0 auto;
+  text-shadow: 3px 3px 5px $darkBlue;
+}
+
+.text {
+  margin: 0;
+}
+
+.navBar {
+  position: absolute;
+}
+
+.navButton {
+  background: none;
+  border: none;
+  outline: none;
+}
+
+.selected {
+  filter: drop-shadow(0px 0px 5px $salmon);
+  transition: all .24s ease-in-out;
+}
+
+.icon {
+  height: 60px; 
+  filter: drop-shadow(0px 0px 5px $darkBlue);
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+@media screen and (max-width: 992px) {
```

Comment:
Really like how you did the media queries

Reply:
Thank you for finding these breakpoints!

Reply:
I agree- the header looks so great in all these different views. Well designed!